packages/edit-site/src/components/page-pages/index.js

```diff
@@ -47,6 +47,10 @@ const defaultConfigPerViewType = {
 		mediaField: 'featured-image',
 		primaryField: 'title',
 	},
+	[ LAYOUT_LIST ]: {
+		primaryField: 'title',
+		preview: true,
```

Comment:
This is presuming the list view can work with and without previews. If we only envision it working with previews, this would be no longer necessary.

Reply:
I ran this by some people, and the common believe is that the list view doesn't make much sense without the preview. For that kind of "row condensed" layout, the table would be better. I'm removing this.